velox/functions/prestosql/aggregates/MinMaxAggregates.cpp

```diff
@@ -417,18 +429,39 @@ class NonNumericMinMaxAggregateBase : public exec::Aggregate {
       if (decoded.isNullAt(i)) {
         return;
       }
+
+      if (shouldCheckNestedNull_) {
+        checkNestedNull(decoded, i, decoded.base()->typeKind());
+      }
+
       if (!accumulator->hasValue() ||
           compareTest(compare(accumulator, decoded, i))) {
         accumulator->write(baseVector, indices[i], allocator_);
       }
     });
   }
+
+  FOLLY_ALWAYS_INLINE static void checkNestedNull(
+      const DecodedVector& decoded,
+      vector_size_t index,
+      TypeKind typeKind) {
```

Comment:
This function doesn't need to take 'typeKind' parameter, it can figure it our on its own.

Reply:
Fixed